atomic-reactor.spec

```diff
@@ -86,6 +69,23 @@ BuildRequires:  python3-six
 BuildRequires:  python3-osbs >= 0.48
 BuildRequires:  python3-responses
 %endif # with_check
+%else
+BuildRequires:  python2-devel
+BuildRequires:  python-setuptools
+%if 0%{?with_check}
+BuildRequires:  pytest
+BuildRequires:  python-dockerfile-parse >= 0.0.11
+%if (0%{?fedora} >= 29)
+BuildRequires:  python2-docker
+%else
+BuildRequires:  python-docker-py
+%endif  # fedora 29+
+BuildRequires:  python-flexmock >= 0.10.2
+BuildRequires:  python-six
+BuildRequires:  python-osbs >= 0.48
+BuildRequires:  python-backports-lzma
+BuildRequires:  python2-responses
+%endif # with_check
 %endif # with_python3
```

Comment:
Could we aggregate the with_check blocks in a single one?

Reply:
ok